solo-chains/node/tanssi-relay-service/src/dev_service.rs

```diff
@@ -108,7 +109,10 @@ struct DevDeps<C, P> {
     /// Manual seal command sink
     pub command_sink: Option<futures::channel::mpsc::Sender<EngineCommand<Hash>>>,
     /// Channels for dev rpcs
-    pub dev_rpc_data: Option<flume::Sender<Vec<u8>>>,
+    pub dev_rpc_data: (
+        Option<flume::Sender<Vec<u8>>>, //downward
+        Option<flume::Sender<Vec<u8>>>, //upward
+    ),
```

Comment:
Give proper names to the channels, or use this struct that already has names:

```suggestion
    pub dev_rpc_data: Option<DevRpc>,
```

Also I don't understand why use an Option here, can we remove it?